src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  font-family: Roboto, sans-serif;
+
+  --blue-word: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  display: block;
+}
+
+.nav {
+  padding: 10px 0;
+}
+
+.nav__items {
+  padding: 0;
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__item {
+  text-align: center;
+  list-style-type: none;
+  margin: 0 10px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 22.5px 0;
+  color: #000;
+}
+
+.nav__link:hover {
+  color: var(--blue-word);
+}
+
+[data-qa='hover']:hover {
+  color: var(--blue-word);
+}
```

Comment:
The hover effect for elements with the data attribute 'data-qa="hover"' is the same as the hover effect for '.nav__link'. Ensure that this duplication is necessary or consider consolidating these styles if they are intended to be the same.